packages/replay/MIGRATION.md

```diff
@@ -0,0 +1,51 @@
+# Upgrading Replay from 0.6.x to 7.24.0
+
+## Replay sample rates are defined on top level (https://github.com/getsentry/sentry-javascript/issues/6351)
+
+Instead of defining the sample rates on the integration like this:
+
+```js
+Sentry.init({
+  dsn: '__DSN__',
+  integrations: [
+    new Replay({
+      sessionSampleRate: 0.1,
+      errorSampleRate: 1.0,
+    })
+  ],
+  // ...
+});
+```
+
+They are now defined on the top level of the SDK:
+
+```js
+Sentry.init({
+  dsn: '__DSN__',
+  replaysSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+  integrations: [
+    new Replay({
+      // other replay config still goes in here
+    })
+  ],
+});
+```
+
+Note that the definition inside of `new Replay({})` has been deprecated and will be removed in a future update.
```

Comment:
l: I think "definition" might be confusing to users. Should we say something like
```suggestion
Note that the sample rate options inside of `new Replay({})` have been deprecated and will be removed in a future update.
```

Reply:
Makes sense!